dcc-portal-ui/tasks/utils/chrome.applescript

```diff
@@ -0,0 +1,39 @@
+on run argv
```

Comment:
When the local dev tasks runs, this opens a new window if one hasn't been opened previously, or focuses on the previously opened tab (instead of opening a new tab)